### PR TITLE
task: add lifecycle timelines and recovery to the logs console

### PR DESCRIPTION
**Pull request by ndizazzo**
## What changes

The logs console now presents one searchable operational ledger with lifecycle timelines, caller and mesh attribution, retained payload inspection, live-stream recovery, cleanup controls, and schema compatibility guidance.

This is part 2 of the replacement stack for #1403 and is based on backend PR #1440.

## Highlights

- Combines request and audit history with filters, event-volume charts, pagination, and reconnect recovery.
- Adds request lifecycle, routing, stream, usage, caller identity, and retained payload inspection.
- Adds accessible cleanup and deletion flows with explicit previews and safe mobile action ordering.
- Keeps navigation compact below 1024px so tablet layouts do not crowd or truncate controls.

### Unified ledger

Unified logs ledger

### Mobile request inspector

Mobile request inspector

### Mobile cleanup review

Mobile cleanup review

The complete nine-capture desktop, tablet, and mobile set is in the [visual QA gist](https://gist.github.com/ndizazzo/4fd67c7859df8be6c5b598f866131950).

## Validation

- `pnpm run lint`
- `pnpm run typecheck`
- Vitest: 1,632 passed, 3 skipped
- Playwright: 63 passed, 2 intentional live-environment skips
- Production UI build
- Root `just build`
- Fresh visual QA at 375px, 768px, and 1280px across nine states; both independent reviewers passed with no blockers

## Reconstruction audit

All 144 original UI paths are present. The final PR has 147 UI-only paths because the responsive review added three correction paths outside the original manifest. Of the original paths, only the footer regression assertion differs from the old head, matching the corrected shared modal behavior. No implementation or test-coverage gap was found.

## Summary by CodeRabbit

* **New Features**
  * Added responsive request lifecycle timelines with pagination, elapsed-time details, and accessibility support.
  * Added SSE payload viewing with frame navigation, JSON formatting, copying, and safe plaintext handling.
  * Added richer caller, peer, audit, and command-summary details.
  * Added chart bucket selection, time-window filtering, and route exclusions for log queries and cleanup previews.
  * Added clearer live recovery, fallback-polling status, reusable pagination, banners, badges, and scrolling improvements.

* **Bug Fixes**
  * Improved responsive layouts, overflow handling, pagination clamping, retained data, and scrollbar stability.

### CODE REVIEW

**Comment by ndizazzo** [comment]
@coderabbitai review

**Comment by ndizazzo** [comment]
@coderabbitai review

File: crates/mesh-llm-ui/src/features/logs/api/client.ts [hunk 1]
```diff
@@ -436,6 +443,7 @@ export class LogsApiClient {
         from: request.from,
         to: request.to,
         route: request.route,
+        excludeRoute: request.excludeRoute,
```
Comment (ndizazzo) [inline crates/mesh-llm-ui/src/features/logs/api/client.ts]: This field is always populated by the cleanup UI, but the backend on the stacked base rejects unknown cleanup fields. A console talking to that host gets a 400 before it can even preview cleanup. Please gate this on a cleanup capability/version, or keep the request compatible with the older body.

File: crates/mesh-llm-ui/src/features/logs/components/EventsOverTimeChart.tsx [hunk 2]
```diff
@@ -121,19 +128,79 @@ export function EventsOverTimeChart({
...
+      if (!onBucketSelect || bucket === undefined) return
+      onBucketSelect({
+        from: new Date(bucket.bucketStart).toISOString(),
+        to: new Date(bucket.bucketEnd).toISOString()
+      })
+    },
+    [data, onBucketSelect]
+  )
+  const handleChartClick = useCallback(
+    (nextState: MouseHandlerDataParam) => selectBucket(bucketIndexOf(nextState) ?? hoveredIndex ?? activeOptionIndex),
```
Comment (ndizazzo) [inline crates/mesh-llm-ui/src/features/logs/components/EventsOverTimeChart.tsx]: Clicking an empty bucket falls through to the last hovered or keyboard-selected bucket here. The page can end up filtering to a different time range than the one the user clicked. An empty bucket should no-op or use its own raw window, not reuse previous selection state.

File: crates/mesh-llm-ui/src/features/logs/components/EventsOverTimeChart.tsx [hunk 3]
```diff
@@ -121,19 +128,79 @@ export function EventsOverTimeChart({
...
+    [bucketIndexOf]
+  )
+  const handleChartMouseLeave = useCallback(() => setHoveredIndex(undefined), [])
+  const selectBucket = useCallback(
+    (index: number | undefined) => {
+      const bucket = index === undefined ? undefined : data[index]
+      if (!onBucketSelect || bucket === undefined) return
+      onBucketSelect({
+        from: new Date(bucket.bucketStart).toISOString(),
+        to: new Date(bucket.bucketEnd).toISOString()
```
Comment (ndizazzo) [inline crates/mesh-llm-ui/src/features/logs/components/EventsOverTimeChart.tsx]: `bucketEnd` is the start of the next bucket, but the log query treats `to` as inclusive. An event exactly on the boundary is drawn in the next bar and also returned for the selected previous bar. Please make the selection half-open, or adjust the upper bound before sending it.

**Review by ndizazzo — COMMENTED** [review]
Needs revision. The cleanup compatibility and chart selection issues need another pass before merge.

Follow-ups:
- Add coverage for a console talking to the stacked backend API, including strict cleanup request parsing.
- Add click tests for empty buckets and events exactly on a bucket boundary.
- The remaining Linux inference smoke was still running during review, so please recheck it after the fixes.

**Review by i386 — COMMENTED** [review]
Reviewed the console slice structurally — the shape is right and I have no blocking concerns, but I'm holding approval until the stack's gates close rather than approving on a pending CI run.

What I checked:

- Stacked on #1440 as documented; the backend files it touches (`cleanup.rs`, `parse.rs`, `maintenance/*`) are additive refinements over that branch, not divergence.
- Coverage is unusually good for a UI PR of this size: 1,632 Vitest cases, 63 Playwright specs including dedicated a11y (`logs-a11y.spec.ts`) and chart-stability specs, plus fixture-driven request-inspector suites.
- Cleanup/deletion flows expose explicit previews and safe mobile action ordering per the body — matching the backend's preview/receipt contract from #1440, and the trusted-local framing of the delete API is preserved (the UI never receives or shows filesystem paths).
- Visual QA across 375/768/1280px with two independent reviewers is documented in the gist.

Follow-ups once CI completes and #1440 merges:

1. Re-run the Linux lane on the rebased head (one job still pending at review time) — happy to approve then.
2. Non-blocking: `request-inspector-fixtures.ts` and `logs-chart-stability.spec.ts` are new long-lived fixtures; worth a header comment pointing at the backend DTO definitions they mirror so they don't silently drift when #1440's DTOs evolve.

File: crates/mesh-llm-ui/e2e/logs/request-inspector-fixtures.ts [hunk 4]
```diff
@@ -4,7 +4,8 @@ export const REQUEST_INSPECTOR_IDS = {
   empty: '10000000-0000-4000-8000-000000000003',
   failed: '10000000-0000-4000-8000-000000000004',
   active: '10000000-0000-4000-8000-000000000005',
-  transient: '10000000-0000-4000-8000-000000000006'
+  transient: '10000000-0000-4000-8000-000000000006',
+  streaming: '10000000-0000-4000-8000-000000000007'
```
Comment (i386) [inline crates/mesh-llm-ui/e2e/logs/request-inspector-fixtures.ts:8]: Inline follow-up to my review comment: these fixtures mirror the backend DTOs introduced in #1440 (base branch). Worth a short header comment naming the specific DTO/route definitions each fixture mirrors, so a backend DTO change shows up as an obvious fixture drift instead of a silent e2e pass against stale shapes.
